snuba/schemas.py

```diff
@@ -14,14 +21,22 @@ def get_time_series_extension_properties(
             "from_date": {
                 "type": "string",
                 "format": "date-time",
-                "default": lambda: (
-                    datetime.utcnow().replace(microsecond=0) - default_window
-                ).isoformat(),
+                "default": track_calls(
+                    timeseries_metrics,
+                    "from_date.default",
```

Comment:
This becomes `snuba.extensions.timeseries.from_date.default` when fully expanded.